packages/components/src/navigable-container/test/index.js

```diff
@@ -0,0 +1,467 @@
+/**
```

Comment:
Any reason why the NavigableMenu and TabbableContainer tests are merged into a single file?

Reply:
Not particularly. I think the main issue is that there's one folder, but actually two different components being exported — and so I didn't know which approach to take.

I'll split the tests into two separate files

Reply:
Just a thought — did we not recently make some changes to some config where it would only consider `index.[ts]s{x}` files, either for storybook or test files? Would that matter for this PR?

Reply:
Yes, for test/story files, `.js` files are excluded from type checking:

https://github.com/WordPress/gutenberg/blob/9620bae6fef4fde7cc2b7833f416e240207cda29/packages/components/tsconfig.json#L39-L40

Reply:
Done in https://github.com/WordPress/gutenberg/pull/43606/commits/d1455f7908f70b826b4ca01dd6eed654f9bdbc5b

Reply:
> Just a thought — did we not recently make some changes to some config where it would only consider `index.[ts]s{x}` files, either for storybook or test files? Would that matter for this PR?

That's actually what I had in mind — these changes proposed in the [update to Storybook V7](https://github.com/WordPress/gutenberg/pull/42486#discussion_r922917992)